### PR TITLE
Feat/faqs page

### PR DESCRIPTION
**Pull request by JM066**
Card components refactored and some of the layout of tablet and mobile version fixed

### CODE REVIEW

File: src/components/Caption/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import classNames from "classnames";
+
+import styles from "./Caption.module.scss";
+
+export interface CaptionProps {
+  header?: JSX.Element;
+  title: JSX.Element;
+  className?: string;
+  color?: "green" | "sky" | "blue";
+  textSize?: "small" | "medium" | "costume";
```
Comment (BruceJi7) [inline src/components/Caption/index.tsx:10]: Costume, is that meant to be custom?

File: src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -47,13 +47,13 @@ function Navbar() {
...
-        <ConunLogo className={styles.ConunLogo} />
         <Link href="/">
           <a>
-            <div className={styles.NetworkCircle}></div>
-            <div>Main net is live</div>
+            <ConunLogo className={styles.ConunLogo} />
           </a>
         </Link>
+        <div className={styles.NetworkCircle}></div>
+        <div>Main net is live</div>
```
Comment (BruceJi7) [inline src/components/Navbar/index.tsx:56]: This should say mainnet with no space, something to change in next PR perhaps lol

**Review by BruceJi7 — APPROVED** [review]

**Comment by BruceJi7** [comment]
The new card components looks easier to follow ! Nice

File: src/components/Caption/index.tsx (hunk 1 above)
Comment (JM066) [reply src/components/Caption/index.tsx:10]: I know.. ther is no consistency in terms of font size. It looks ugly. I will see what I can do

File: src/components/Navbar/index.tsx (hunk 2 above)
Comment (JM066) [reply src/components/Navbar/index.tsx:56]: Actually, that's what Aziz kept telling me. haha! I will correct it

**Pull request merged**